docs/UserGuide.md

```diff
@@ -501,4 +544,62 @@ Example of usage:
+| List all exercises     			| exercise /list       								     | exercise /list     			       |
+| Search exercise    	 			| exercise /search <EXERCISE_NAME [string]>     				     | exercise /search pushups      		       |
+| Add a workout plan     			| workout /create <WORKOUT_PLAN_NAME [string]>       				     | workout /create push day      		       |
+| Delete a workout plan  			| workout /delete <WORKOUT_PLAN_NAME [string]>       				     | workout /delete push day      		       |
+| List workout plan     	 		| workout /list        								     | workout /list       			       |
+| Search workout plans     	 		| workout /search <WORKOUT_PLAN_NAME [string]>       				     | workout /search push day      		       |
+| Assign an exercise to a workout plan  	| workout /assign <EXERCISE_NAME [string]> /to <WORKOUT_PLAN_NAME [string]>          | workout /assign pushups /to push day            |
+| Remove an exercise from a workout plan     	| workout /unassign <EXERCISE_NAME [string]> /from <WORKOUT_PLAN_NAME [string]>      | workout /unassign pushups /from push day        |
+| List all exercises in a workout plan     	| workout /info <WORKOUT_PLAN_NAME [string]>       				     | workout /info push day      		       |
+| Choose a workout plan for a day     	 	| program /assign <WORKOUT_PLAN_NAME [string]> /to <DAY [string]>       	     | program /assign push day /to monday             |
+| View Today's workout plan    	 		| program /today       								     | program /today      			       |
+| View Weekly workout plan     	 		| program /list       								     | program /list      			       |
+| Remove a workout plan for a day     	 	| program /clear <DAY [string]>      						     | program /clear Tuesday      		       |
+| Adding an exercise log    			| program /log <EXERCISE_NAME [string]> /weight <WEIGHT [integer]> /sets <NUMBER_OF_SETS [integer]> /reps <NUMBER_OF_REPS [integer]>   | program /log benchpress /weight 125 /sets 3 /reps 5    |
+| Adding an exercise log for a separate date    			| program /log <EXERCISE_NAME [string]> /weight <WEIGHT [integer]> /sets <NUMBER_OF_SETS [integer]> /reps <NUMBER_OF_REPS [integer]> /date <DATE [yyyy-mm-dd]>   | program /log benchpress /weight 125 /sets 3 /reps 5 /date 2024-03-25    |
+| Viewing historic logs     	 		| program /history      							     | program /history      			       |
+| Viewing historic logs     	 		| program /history <DATE [yyyy-mm-dd]>      					     | program /history <DATE [yyyy-mm-dd]>   	       |
+| Displaying Help Menu Category: Exercise    	| help /exercise <INDEX [integer]>       					     | help /exercise 3     			       |
+| Displaying Help Menu Category: Workout    	| help /workout  <INDEX [integer]>     						     | help /workout 4     			       |
+| Displaying Help Menu Category: Program    	| help /program  <INDEX [integer]>        					     | help /program 7      			       |
+
+
```

Comment:
Great additions to UG!

Reply:
oops I forgot about the help menu link! Thanks for the reminder